tests/components/FilterSearch.test.tsx

```diff
@@ -174,12 +176,15 @@ describe('search with section labels', () => {
         expect(setFilterOption).toBeCalledWith(expectedSetFilterOptionParam);
       });
       expect(setOffset).toBeCalledWith(expectedSetOffsetParam);
+      expect(resetFacets).toBeCalled();
 
       const setFilterOptionCallOrder = setFilterOption.mock.invocationCallOrder[0];
       const setOffsetCallOrder = setOffset.mock.invocationCallOrder[0];
+      const resetFacetsCallOrder = resetFacets.mock.invocationCallOrder[0];
       const mockExecuteSearchCallOrder = mockExecuteSearch.mock.invocationCallOrder[0];
       expect(setFilterOptionCallOrder).toBeLessThan(mockExecuteSearchCallOrder);
       expect(setOffsetCallOrder).toBeLessThan(mockExecuteSearchCallOrder);
+      expect(resetFacetsCallOrder).toBeLessThan(mockExecuteSearchCallOrder);
     });
```

Comment:
it seems like `toHaveBeenCalledBefore` is only available for special types of mocked functions. when I try to use it, I get an error that the property doesn't exist on type `JestMatchers<Mock<any, any>>`. casting the mocked function to a specific type instead of `any` doesn't help either

Reply:
actually, this might have been part of `jest-extended` package, which we don't have it installed.